# Belarusian translation for ubuntuone-client
# Copyright (c) 2009 Rosetta Contributors and Canonical Ltd 2009
# This file is distributed under the same license as the ubuntuone-client package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2009.
#
msgid ""
msgstr ""
"Project-Id-Version: ubuntuone-client\n"
"Report-Msgid-Bugs-To: FULL NAME <EMAIL@ADDRESS>\n"
"POT-Creation-Date: 2012-09-10 21:26+0000\n"
"PO-Revision-Date: 2011-10-22 21:30+0000\n"
"Last-Translator: Soltan Dzmitry <Unknown>\n"
"Language-Team: Belarusian <be@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && "
"n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"
"X-Launchpad-Export-Date: 2012-09-17 18:32+0000\n"
"X-Generator: Launchpad (build 15966)\n"

#: ../ubuntuone/status/aggregator.py:51
msgid "Magicicada"
msgstr "Magicicada"

#: ../ubuntuone/status/aggregator.py:52
msgid "New cloud folder(s) available"
msgstr "Даступна новая воблачная тэчка (кі)"

#: ../ubuntuone/status/aggregator.py:53
msgid "File synchronization completed."
msgstr "Сінхранізацыя файлаў завершана."

#: ../ubuntuone/status/aggregator.py:55
#, python-format
msgid "%(percentage_completed)d%% completed."
msgstr "%(percentage_completed)d%% завершана."

#: ../ubuntuone/status/aggregator.py:56
msgid "File synchronization in progress"
msgstr "Выконваецца сінхранізацыя файлаў"

#: ../ubuntuone/status/aggregator.py:59
#, python-format
msgid ""
"There is no available space on the folder:\n"
"\"%s\" shared by %s"
msgstr ""
"Няма вольнага месца ў тэчцы:\n"
"«%s» адкрыў супольны доступ: %s"

#: ../ubuntuone/status/aggregator.py:73
#, python-format
msgid "'%(filename)s' is being uploaded to your personal cloud."
msgstr "'%(filename)s' загружаецца ў вашае асабістае воблака."

#: ../ubuntuone/status/aggregator.py:79
#, python-format
msgid ""
"'%(filename)s' and %(other_files)d other file are being uploaded to your "
"personal cloud."
msgid_plural ""
"'%(filename)s' and %(other_files)d other files are being uploaded to your "
"personal cloud."
msgstr[0] ""
"«%(filename)s» і %(other_files)d іншы файл загружаюцца ў вашае асабістае "
"воблака."
msgstr[1] ""
"«%(filename)s» і %(other_files)d іншых файла загружаюцца ў вашае асабістае "
"воблака."
msgstr[2] ""
"«%(filename)s» і %(other_files)d іншых файлаў загружаюцца ў вашае асабістае "
"воблака."

#: ../ubuntuone/status/aggregator.py:90
#, python-format
msgid "'%(filename)s' is being downloaded to your computer."
msgstr "'%(filename)s' загружаецца на ваш кампутар."

#: ../ubuntuone/status/aggregator.py:96
#, python-format
msgid ""
"'%(filename)s' and %(other_files)d other file are being downloaded to your "
"computer."
msgid_plural ""
"'%(filename)s' and %(other_files)d other files are being downloaded to your "
"computer."
msgstr[0] ""
"'%(filename)s' і %(other_files)d іншы файл загружаюцца на ваш кампутар."
msgstr[1] ""
"'%(filename)s' і %(other_files)d іншых файла загружаюцца на ваш кампутар."
msgstr[2] ""
"'%(filename)s' і %(other_files)d іншых файлаў загружаюцца на ваш кампутар."

#: ../ubuntuone/status/aggregator.py:107
#, python-format
msgid "'%(filename)s' was uploaded to your personal cloud."
msgstr "'%(filename)s' загружаны ў вашае асабістае воблака."

#: ../ubuntuone/status/aggregator.py:113
#, python-format
msgid ""
"'%(filename)s' and %(other_files)d other file were uploaded to your personal "
"cloud."
msgid_plural ""
"'%(filename)s' and %(other_files)d other files were uploaded to your "
"personal cloud."
msgstr[0] ""
"'%(filename)s' і %(other_files)d іншы файл былі загружаныя ў вашае асабістае "
"воблака."
msgstr[1] ""
"'%(filename)s' і %(other_files)d іншых файла былі загружаныя ў вашае "
"асабістае воблака."
msgstr[2] ""
"'%(filename)s' і %(other_files)d іншых файлаў былі загружаныя ў вашае "
"асабістае воблака."

#: ../ubuntuone/status/aggregator.py:124
#, python-format
msgid "'%(filename)s' was downloaded to your computer."
msgstr "'%(filename)s' быў загружаны на ваш кампутар."

#: ../ubuntuone/status/aggregator.py:130
#, python-format
msgid ""
"'%(filename)s' and %(other_files)d other file were downloaded to your "
"computer."
msgid_plural ""
"'%(filename)s' and %(other_files)d other files were downloaded to your "
"computer."
msgstr[0] ""
"'%(filename)s' і %(other_files)d іншы файл былі загружаныя на ваш кампутар."
msgstr[1] ""
"'%(filename)s' і %(other_files)d іншых файла былі загружаныя на ваш кампутар."
msgstr[2] ""
"'%(filename)s' і %(other_files)d іншых файлаў былі загружаныя на ваш "
"кампутар."

#: ../ubuntuone/status/aggregator.py:191
#, python-format
msgid "A file was just made public at %(new_public_url)s"
msgstr "Файл быў апублікаваны па адрасе %(new_public_url)s"

#: ../ubuntuone/status/aggregator.py:204
#, python-format
msgid "%(event_count)d file was just made public."
msgid_plural "%(event_count)d files were just made public."
msgstr[0] "%(event_count)d файл апублікаваны."
msgstr[1] "%(event_count)d файлаў апублікаваныя."
msgstr[2] "%(event_count)d файлаў апублікавана."

#: ../ubuntuone/status/aggregator.py:212
msgid "A file is no longer published"
msgstr "Адкрыты доступ да файла зачынены"

#: ../ubuntuone/status/aggregator.py:220
#, python-format
msgid "%(event_count)d file is no longer published."
msgid_plural "%(event_count)d files are no longer published."
msgstr[0] "Адкрыты доступ да %(event_count)d файла зачынены"
msgstr[1] "Адкрыты доступ да %(event_count)d файлаў зачынены"
msgstr[2] "Адкрыты доступ да %(event_count)d файлаў зачынены"

#: ../ubuntuone/status/aggregator.py:235
#, python-format
msgid "Found %(event_count)d new cloud folder."
msgid_plural "Found %(event_count)d new cloud folders."
msgstr[0] "Знойдзена %(event_count)d воблачная тэчка."
msgstr[1] "Знойдзена %(event_count)d воблачныя тэчкі."
msgstr[2] "Знойдзена %(event_count)d воблачных тэчак."

#: ../ubuntuone/status/aggregator.py:243
#, python-format
msgid ""
"New cloud folder available: '%(folder_name)s' shared by %(other_user_name)s"
msgstr ""
"Даступна новая тэчка ў воблаку: «%(folder_name)s» апублікавана "
"%(other_user_name)s"

#: ../ubuntuone/status/aggregator.py:259
#, python-format
msgid "New cloud folder available: '%(folder_name)s'"
msgstr "Новая тэчка ў воблаку даступна: '%(folder_name)s'"

#: ../ubuntuone/status/aggregator.py:281
msgid "The connection to the server was lost."
msgstr "Злучэнне з серверам страчана."

#: ../ubuntuone/status/aggregator.py:287
msgid "The connection to the server was restored."
msgstr "Злучэнне з серверам адноўлена."

#: ../data/emblem-ubuntuone-downloading.icon.in.h:1
msgid "Downloading"
msgstr "Загрузка"

#: ../data/emblem-ubuntuone-unsynchronized.icon.in.h:1
msgid "Unsynchronized"
msgstr "Не сінхранізавана"

#: ../data/emblem-ubuntuone-uploading.icon.in.h:1
msgid "Uploading"
msgstr "Дасыланне дадзеных"
